x-pack/plugins/infra/server/lib/alerting/metric_threshold/register_metric_threshold_rule_type.ts

```diff
@@ -191,7 +193,10 @@ export async function registerMetricThresholdRuleType(
     },
     category: DEFAULT_APP_CATEGORIES.observability.id,
     producer: 'infrastructure',
-    alerts: MetricsRulesTypeAlertDefinition,
+    alerts: {
+      ...MetricsRulesTypeAlertDefinition,
+      shouldWrite: true,
```

Comment:
We added this flag to the framework registration because we didn't want rules that were registered with both the rule registry and the alerting framework to have both write alerting docs. We should be able to remove this once we move all rules away from the rule registry